prowler/lib/outputs/compliance/models.py

```diff
@@ -4,9 +4,9 @@
 
 
 # TODO: move this to outputs/<compliance>/models.py
-class Check_Output_MITRE_ATTACK(BaseModel):
+class Check_Output_MITRE_ATTACK_AWS(BaseModel):
```

Comment:
Could you move this to `prowler/lib/outputs/compliance/mitre/models.py`?

Reply:
Done